plutus-benchmark/validation/bench/BenchCekPAPI.hs

```diff
@@ -0,0 +1,39 @@
+{- | Validation benchmarks for the CEK machine. -}
+{-# LANGUAGE BangPatterns #-}
+module Main where
+
+import Common
+import Control.Exception (evaluate)
+import Data.Proxy (Proxy (..))
+import PlutusBenchmark.Common (toNamedDeBruijnTerm)
+import PlutusCore.Default (BuiltinSemanticsVariant (DefaultFunSemanticsVariantA))
+import PlutusLedgerApi.Common (PlutusLedgerLanguage (PlutusV1))
+import System.Directory (listDirectory)
+import Test.Tasty (askOption, defaultMainWithIngredients, includingOptions, testGroup)
+import Test.Tasty.Options (OptionDescription (..))
+import Test.Tasty.PAPI (benchIngredients)
+import UntypedPlutusCore as UPLC
+
+{-|
+ Benchmarks only for the CEK execution time of the data/*.flat validation scripts
+
+ Run the benchmarks.  You can run groups of benchmarks by typing things like
+     `stack bench -- plutus-benchmark:validation --ba crowdfunding`
+   or
+     `cabal bench -- plutus-benchmark:validation --benchmark-options crowdfunding`.
+-}
+
+main :: IO ()
+main = do
+  scriptDirectory <- getScriptDirectory
+  files <- listDirectory scriptDirectory
+  evalCtx <- evaluate $ mkEvalCtx PlutusV1 DefaultFunSemanticsVariantA
+  let customOpts  = [Option (Proxy :: Proxy QuickFlag)]
+      ingredients = includingOptions customOpts : benchIngredients
+
+      mkCekBM file program =
+          benchTermCek evalCtx . toNamedDeBruijnTerm . UPLC._progTerm $ unsafeUnflat file program
```

Comment:
Hm, why are we even including the cost of `unflat` here? And in the original `validation` benchmarks. That should be discussed separately, not a blocker for this PR.